packages/__examples__/.storybook/preview.js

```diff
@@ -48,54 +35,9 @@ addParameters({
       name: 'instructure-ui',
       brandUrl: 'https://instructure.design',
       brandImage:
-        'https://instructure.design/icons-bea652a7d0546b99c7cba9d3b4a538ee/favicon-32x32.png'
+        'https://instructure.design/icons-bea652a7d0546b99c7cba9d3b4a538ee/favicon-32x32.png' // TODO this URL is broken
     }),
     showPanel: false,
     isFullscreen: false
   }
 })
-
-configure(() => {
-  const examplesContext = require.context(
-    '../../',
-    true,
-    /^.*\/src\/.*\.examples\.js$/,
-    'lazy'
```

Comment:
This was one of the things causing issues with Storybook 6. We were loading our modules lazily with a promise (this is some barely documented Webpack feature) that caused Storybook to throw errors saying "Error: Cannot add a story when not configuring ...". I've set this to 'sync', see stories.js